aidants_connect/settings.py

```diff
@@ -557,44 +558,33 @@ def getenv_bool(key: str, default: Optional[bool] = None) -> bool:
     Je vous contacte car je ne reçois pas les emails de confirmation de mon adresse email.""",
 )
 
-EMAIL_HABILITATION_ISSUER_EMAIL_ALREADY_EXISTS_FROM = os.getenv(
-    "EMAIL_HABILITATION_ISSUER_EMAIL_ALREADY_EXISTS_FROM", SUPPORT_EMAIL
+EMAIL_ORGANISATION_REQUEST_FROM = os.getenv(
+    "EMAIL_ORGANISATION_REQUEST_FROM", SUPPORT_EMAIL
 )
 
 EMAIL_HABILITATION_ISSUER_EMAIL_ALREADY_EXISTS_SUBJECT = os.getenv(
     "EMAIL_HABILITATION_ISSUER_EMAIL_ALREADY_EXISTS_SUBJECT",
-    "Rappel de votre profil demandeur",
-)
-
-EMAIL_ORGANISATION_REQUEST_CREATION_FROM = os.getenv(
-    "EMAIL_ORGANISATION_REQUEST_CREATION_FROM", SUPPORT_EMAIL
+    "Aidants Connect - Rappel de votre profil demandeur",
 )
 
 EMAIL_ORGANISATION_REQUEST_CREATION_SUBJECT = os.getenv(
     "EMAIL_ORGANISATION_REQUEST_CREATION_SUBJECT",
-    "Votre demande d’habilitation Aidants Connect a été créée",
+    "Aidants Connect - Votre demande d’habilitation a été créée",
 )
 
-EMAIL_ORGANISATION_REQUEST_MODIFICATION_FROM = os.getenv(
-    "EMAIL_ORGANISATION_REQUEST_MODIFICATION_FROM", SUPPORT_EMAIL
+EMAIL_ORGANISATION_REQUEST_SUBMISSION_SUBJECT = os.getenv(
+    "EMAIL_ORGANISATION_REQUEST_CREATION_SUBJECT",
+    "Aidants Connect - Votre demande d’habilitation a été soumise",
 )
 
 EMAIL_ORGANISATION_REQUEST_MODIFICATION_SUBJECT = os.getenv(
     "EMAIL_ORGANISATION_REQUEST_MODIFICATION_SUBJECT",
-    "Votre demande d’habilitation Aidants Connect a été modifiée",
-)
-
-EMAIL_ORGANISATION_REQUEST_ACCEPTANCE_FROM = os.getenv(
-    "EMAIL_ORGANISATION_REQUEST_ACCEPTANCE_FROM", SUPPORT_EMAIL
-)
-
-EMAIL_NEW_MESSAGE_RECEIVED_FROM = os.getenv(
-    "EMAIL_NEW_MESSAGE_RECEIVED_FROM", SUPPORT_EMAIL
+    "Aidants Connect - Votre demande d’habilitation a été modifiée",
 )
 
 EMAIL_NEW_MESSAGE_RECEIVED_SUBJECT = os.getenv(
     "EMAIL_NEW_MESSAGE_RECEIVED_SUBJECT",
-    "Vous avez reçu un nouveau message d'Aidants Connect",
+    "Aidants Connect - Vous avez reçu un nouveau message de l'équipe Aidants Connect",
```

Comment:
```suggestion
    "Aidants Connect - Vous avez reçu un nouveau message de l’équipe Aidants Connect",
```

(Tant qu'à changer les textes, j'en profite pour rectifier les petits problèmes de typographie)